tox.ini

```diff
@@ -10,22 +10,25 @@ envlist =
 description = check code style
 skip_install = true
 deps =
-    ruff
+    -r requirements/build.txt
```

Comment:
Shouldn't ``ruff`` be specified in ``requirements/test.txt``?

Reply:
Sure, that seems fine to me. Would it be same for bandit? 

To motivate what I did: Since test and prod need to go together, I was just trying to limit the test set and place everything else in build.

Reply:
> I was just trying to limit the test set and place everything else in build.

We could always be completely specific and put ruff and bandit in their own requirements file.

Reply:
any preference here? doing the specific one is maybe more correct while a bit more verbose - but with tox it wouldn't matter to user and folding them into all also wouldn't matter if not using tox, so maybe that?

Reply:
Yeah, I think it is overkill, but if you want separation it's the only way.

I think if I had to flip a coin, I'd just lump ruff and bandit in test.txt and call it a day. At least they're semantically where they belong.

🤷 ?